src/bluecore_api/app/routes/other_resources.py

```diff
@@ -94,9 +94,9 @@ async def create_other_resource(
 ):
     time_now = datetime.now(UTC)
     db_other_resource = OtherResource(
-        data=json.loads(resource.data),
         uri=resource.uri,
         is_profile=resource.is_profile,
+        data=json.loads(resource.data),
```

Comment:
`is_profile` needs to be set prior to `data` for the automatic JSON-LD framing to work correctly. There are similar changes below.